test/Bounty.test.js

```diff
@@ -1,11 +1,32 @@
 
 let sendReward = function (sender, receiver, value) {
-  web3.eth.sendTransaction({
-    from: sender,
-    to: receiver,
-    value: value,
+  return new Promise(function (accept, reject) {
```

Comment:
please run `npm run lint:all:fix` — our linter rules will enforce that the arguments to the promise function are called `resolve` and `reject`